airflow/metrics/otel_logger.py

```diff
@@ -83,10 +95,51 @@ def name_is_otel_safe(prefix: str, name: str) -> bool:
     return bool(stat_name_otel_handler(prefix, name, max_length=OTEL_NAME_MAX_LENGTH))
 
 
+def _type_as_str(obj: Instrument) -> str:
+    """
+    Given an OpenTelemetry Instrument, returns the type of the instrument as a string.
+
+    :param obj: An OTel Instrument or subclass
+    :returns: The type() of the Instrument without all the nested class info
+    """
+    # type() will return something like: <class 'opentelemetry.sdk.metrics._internal.instrument._Counter'>
+    # This extracts that down to just "Counter" (or "Gauge", etc.) for cleaner logging purposes.
```

Comment:
this is out of date with the new code :p
the only thing of interest to mention now is that class names are prefixed by `_`, which is why the `[1:]` is necessary

Reply:
fixed, thanks.    I'd say it's still kind of accurate, just less relevant :P